OpenRA.Mods.Cnc/Traits/Disguise.cs

```diff
@@ -188,7 +188,7 @@ public void DisguiseAs(Actor target)
 				}
 				else
 				{
-					var tooltip = target.TraitsImplementing<ITooltip>().FirstOrDefault();
+					var tooltip = target.TraitsImplementing<ITooltip>().FirstEnabledTraitOrDefault();
```

Comment:
This doesn't look right to me.
If it's null, we crash immediately on the line below anyway, so in my opinion the `OrDefault` has to go away.

Reply:
> Throwing a NRE is actually more correct, IMO. https://github.com/OpenRA/OpenRA/pull/18604#discussion_r487407515

Reply:
I instead went with the new C# language syntax which inlines null checks.

Reply:
This is going to move the crash to somewhere less obvious (i.e. when something tries to use `AsPlayer` or `AsTooltipInfo`), which would will make it harder to debug the underlying problem - that `DisguiseAs` was called with an invalid target.

If you really want to just squash the PVS error (without redesigning the API to do make this e.g. `bool TryDisguiseAs(Target target)`) the only valid bandaid is to to add a null check and throw a different kind of exception.

Reply:
I thought nulling things will just make the tooltip disappear.